docs/language/pipelines.md

```diff
@@ -169,7 +169,17 @@ Configuration Parameters:
 
 - `interval`: Time interval in nanoseconds after which the window closes.
 
+##### Windows Based on State
 
+The most flexible windows are code or state based windows. Instead of being fixed on the timestamp
+or a message cout it allows to express the window logic of closing and emitting in tremor-script.
```

Comment:
```suggestion
or a message count, it allows to express the window logic of closing and emitting in tremor-script.
```

Reply:
We need to explain the mechanics of `closing` and `emitting`. I think we do not do it anywhere else, and tremor users cannot be expected to understand how this works.

Reply:
Good call on this, I'll rewrite it to omit special terms and make it more user friendly to read